src/App.js

```diff
@@ -2,7 +2,7 @@ import {  Route, Routes } from "react-router-dom";
 import "./App.css";
 import Mockman from "mockman-js";
 
-import {HomePage} from "./index/index";
+import {Footer, HomePage} from "./index/index";
 import {Products} from "./index/index";
 import {Product} from "./index/index";
 import {ErrorPage} from "./index/index";
```

Comment:
You can import in one line like 
import {HomePage,Footer,Products,Product,ErrorPage} from './index/index'

Reply:
Done. Made the changes and committed